core/src/main/resources/filodb-defaults.conf

```diff
@@ -183,6 +183,33 @@ filodb {
       downsample-schema = "delta-histogram"
     }
 
+    prom-histogram-min-max {
+      columns = ["timestamp:ts",
+                 "sum:double:detectDrops=true",
+                 "count:double:detectDrops=true",
+                 "min:double:detectDrops=true",
+                 "max:double:detectDrops=true",
+                 "h:hist:counter=true"]
+      value-column = "h"
+      downsamplers = ["tTime(0)", "dLast(1)", "dLast(2)", "dMin(3)", "dMax(4)", "hLast(5)"]
+      # Downsample periods are determined by counter dips of the count column
+      downsample-period-marker = "counter(2)"
+      downsample-schema = "prom-histogram-min-max"
+    }
+
+    delta-histogram-min-max {
+      columns = ["timestamp:ts",
+        "sum:double:{detectDrops=false,delta=true}",
+        "count:double:{detectDrops=false,delta=true}",
+        "min:double:{detectDrops=false,delta=true}",
```

Comment:
Based on [this](https://opentelemetry.io/docs/specs/otel/metrics/data-model/#histogram) page the min and max fields are optional. For delta this especially means the value for min field can now be negative unlike sum, count and max.
Also, how should we interpret the word “optional”.  Is it guaranteed that once min and max are published they will always be published on each interval (even if same value or delta of 0) or the lack of this field be treated as delta of 0 ( I’m assuming this might be done to minimize data sent over the wire from agents?) in our case these are different schemas and we don’t want to fail queues with multiple schemas found error.

Reply:
Good question @amolnayak311 . From what I understand, we are adding a different schema altogether, which *needs* to have min-max provided by the agent (we are creating a new type/proto altogether to avoid this confusion) and the gateway would then consistently create the required schema, avoiding this issue altogether.

@sherali42